test/mock-payment-request.js

```diff
@@ -58,7 +62,8 @@ const submitPaymentRequest = {
     completedPaymentRequestId: 1
   }],
   paymentRequestId: 1,
-  referenceId: REFERENCE_ID
+  referenceId: REFERENCE_ID,
+  status: COMPLETED
 }
 
 module.exports = {
```

Comment:
Consider moving file to `mock-components` folder

Reply:
I wouldn't want to move it to `mock-components` as they will get lost in that and loss readability and separation of, "these objects are MADE UP of these components"

A `mock-objects` folder could be the way forward if we have many, may of them and they loss that readability.

For now I think becuz they are only 3/4/5 they're fine where they are -- certainly in future if we get 6/7/8+ then organising under a folder will be the way forward